src/components/join-flow/username-step.jsx

```diff
@@ -195,6 +203,9 @@ class UsernameStep extends React.Component {
                                         /* eslint-enable react/jsx-no-bind */
                                     />
                                     <FormikInput
+                                        autoCapitalize="off"
+                                        autoComplete={values.showPassword ? 'off' : 'new-password'}
```

Comment:
same question about the value of autocomplete here.

Reply:
see above